index.html

```diff
@@ -11,55 +11,56 @@
         <p><a class="btn btn-outline-inverse btn-lg" href="/getting-started/" role="button">Get Started</a></p>
       </div>
     </div>
     {% if jekyll.environment == "production" %}<center><div class="horizontal" data-ea-publisher="markdownguide" data-ea-type="image"></div></center>{% endif %}
     <div class="container">
       <div class="md-guide-feature">
-        <h2 class="no-anchor">Everything you need to learn Markdown.</h2>
-        <p class="lead" style="margin-bottom: 60px">Markdown makes writing on the web fast and easy. <em>The Markdown Guide</em> teaches you how to use it.</p>
+        <h2 class="no-anchor">Trucol, decentralising test driven development.</h2>
+        <p class="lead" style="margin-bottom: 60px">The TruCol protocol presents an improvement of market efficiency and developer autonomy by decentralisation and automation of test-driven development.</p>
         <div class="row">
           <div class="col-sm-4">
             <a href="/getting-started/">
               <div class="icon">
                 <i class="fas fa-edit"></i>
               </div>
             </a>
-            <h4 class="no-anchor">Learn the ropes.</h4>
-            <p>Start using Markdown right now by following along with the <a href="/getting-started/">Getting Started</a> guide. It's designed for everyone, even novices.</p>
+            <h4 class="no-anchor">Start using TruCol</h4>
+            <p>Want to start with TruCol immediately ? Check out the <a href="/getting-started/">Getting Started</a> guide. It's designed for a quick and painfree introduction to TruCol Protocl.</p>
           </div>
           <div class="col-sm-4">
             <a href="/basic-syntax/">
               <div class="icon">
                 <i class="fas fa-book-open"></i>
               </div>
             </a>
-            <h4 class="no-anchor">Dive into the syntax.</h4>
-            <p>Whether you're new to Markdown or a seasoned pro, you'll find the answers to your formatting questions on the <a href="/basic-syntax/">basic syntax page</a>.</p>
+            <h4 class="no-anchor">What is TruCol?</h4>
+            <p>Would you like to learn more about the TruCol protocol, please see the documentation on the <a href="/protocol/"> TruCol Protocol page</a>.</p>
           </div>
           <div class="col-sm-4">
             <a href="/extended-syntax/">
               <div class="icon">
-                <i class="fas fa-code"></i>
+                <i class="fas fa-tools"></i>
               </div>
             </a>
-            <h4 class="no-anchor">Take it up a notch.</h4>
-            <p>Make your Markdown documents awesome by using <a href="/extended-syntax/">extended syntax</a> to create tables, fenced code blocks, automatic links, and more.</p>
+            <h4 class="no-anchor">TruCol and You</h4>
+            <p>Not sure whether TruCol is right for you, checkout the <a href="/for-you/">For You? page</a> to see whether TruCol is what you need!</p>
```

Comment:
`Not sure whether TruCol is right for you` could perhaps be rephrased more positively to: `Would you like to know how you can use TruCol, check out..` or `You can see how you can use TruCol at..`.

Reply:
I have changed it to 
```Would you like to know how you can use TruCol, check out..```